gexpect.go

```diff
@@ -71,7 +71,7 @@ func (buf *buffer) ReadRune() (r rune, size int, err error) {
 		if err != nil {
 			return 0, 0, err
 		}
-		if utf8.FullRune(chunk) {
+		if utf8.FullRune(chunk[:l]) {
```

Comment:
yeah, you are completely right here - I made and tested the fix from 93 line, and then shamefully copied it here, without thinking - it definitely should be `n` here